spec/helpers/pre_built_pages_helper_spec.rb

```diff
@@ -39,7 +39,7 @@
     end
     context 'when 2015-03-01' do
       before { Timecop.travel(2015, 3, 1) }
-      it { is_expected.to eq('2015s') }
+      it { p Time.now; is_expected.to eq('2015s') }
```

Comment:
Do not use semicolons to terminate expressions.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.